docs/parameters.md

```diff
@@ -0,0 +1,150 @@
+# Installation Parameters
+
+This document covers installation parameters as presented to users, and includes how the puppet-based `foreman-installer` parameters will map to new parameters.
+
+## Design Principles
+
+* Using a single parameter when there are multiple variables that should have the same value
+* Aim for minimal configuration values for the user
+
+## Mapping
+
+Parameters are split into either mapped or unmapped groupings per category.
+
+Mapped: Parameters the new installer will have and are mapped to a parameter from the `foreman-installer`.
+Unmapped: Parameters from `foreman-installer` that appear in the official [foreman-documentation](https://github.com/theforeman/foreman-documentation) that are unmapped into the new installer.
+
+### Foreman
+
+These are parameters that are related to the install and operation of the Foreman server and plugins.
+
+#### Database
+
+There are multiple use cases from the users perspective that dictate what parameters need to be available for user input.
```

Comment:
I thought it would be good to have context for the choices we make with parameters by calling out the use cases they are intended to support. This may expand the scope of this document or alter it's shape in the long run where parameters are a piece of the bigger picture.